example.js

```diff
@@ -1,4 +1,4 @@
-const libui = require('.');
+const libui = require('./index.js');
```

Comment:
I was testing with Node.js 0.10 and that was the only thing that caused it to fail, so I changed it and forgot to add a note... It seems the automatic index.js fetching is a later feature. Hopefully nobody actually uses such an old version though.